examples/entities.curl

```diff
@@ -1 +1,3 @@
-curl -X POST -H "X-RosetteAPI-Key: your_api_key" -H "Content-Type: application/json" -H "Accept: application/json" -H "Cache-Control: no-cache" -d '{"content": "Bill Murray will appear in new Ghostbusters film: Dr. Peter Venkman was spotted filming a cameo in Boston this… http://dlvr.it/BnsFfS" }' "https://api.rosette.com/rest/v1/entities"
+# to improve performance, and if you don't need the QID, set this option inside json in -d flag
```

Comment:
For the curl examples, don't add the comment.  The examples are pure curl, so the '#" breaks the command.